images/dcos/bootstrap.sh

```diff
@@ -121,7 +118,15 @@ cat <<EOF >${cloud_config}
 EOF
 
 # address of the apiserver
-kube_master="http://${apiserver_host}:${apiserver_port}"
+kube_master="http://${host_ip}:${apiserver_port}"
```

Comment:
Made everything explicit now: `kube_master` is the kube-master. `kube_master_proxy` is the kube-master proxy. Should be clearer.